ksqldb-engine/src/test/java/io/confluent/ksql/query/QueryBuilderTest.java

```diff
@@ -95,7 +95,7 @@
 import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
in addition to having a test for this, WDYT about persisting this information alongside the state store? basically if the state store gets loaded and we've passed in the wrong changelog topic it should fail. we can do this in ksql, but I feel like that'd be a good feature to have in streams in general to prevent this corruption

Reply:
Thats a good idea. So if for some reason you want to change the chagnelog name you would need to wipe the store, which presumably you would want to anyways

Reply:
👍 exactly (though I don't think we should automate that)

Reply:
@agavra can you put that idea in a streams ticket and we can maybe put that as an AI

Reply:
yeah this would be great, technically we already have that in the checkpoint file but we don't always write it ASAP (and delete it during processing for EOS) so I agree this would be very useful

Reply:
one thought: would it make sense to do this within/as part of Streams, or should this be done in the ksql layer? Maybe we should file it as a ksql ticket for now and then we can decide where to put it as part of the ticket scope

Reply:
@wcarlson5 can you file this (unless @agavra  already has?) before you merge this/call it "done"? Just want to make sure this doesn't get lost

Reply:
https://confluentinc.atlassian.net/browse/KSQL-9056